tests/queryComplex.js

```diff
@@ -38,4 +38,10 @@ describe('Complex selector query', function () {
             simpleProgram.body[2]
         ]);
     });
+
+    it('can not match a top level node', function () {
+        // Test fix for issue #135: half of a child selector matches a top-level node.
+        const matches = esquery(simpleProgram, 'NonExistingNodeType > *');
+        assert.includeMembers(matches, []);
```

Comment:
This assert will always pass. `includeMembers` [asserts a subset relationship](https://www.chaijs.com/api/assert/#method_includemembers). I think you were looking to assert emptiness, like this:

```suggestion
        assert.isEmpty(matches);
```